### PR TITLE
Add unique key info in slave proto

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
Add unique key info in slave proto
need by https://github.com/moiot/gravity/issues/251
### What is changed and how it works?
Add unique key info in slave proto

### Check List 

Tests

### CODE REVIEW

**Comment by suzaku** [comment]
Where do we set this extra field?

**Comment by july2993** [comment]
> Where do we set this extra field?

by drainer.

File: tidb-binlog/slave_binlog_proto/proto/binlog.proto [hunk 1]
```diff
@@ -53,6 +53,16 @@ message Table {
...
   repeated TableMutation mutations = 4;
+  // will only be set with version >= 3.0.9
+  repeated Key unique_keys = 5;
+}
+
+
+// Key contains Key info.
+message Key {
+    // name will be PRIMARY if it's the primary key.
+	optional string name = 1;
```
Comment (suzaku) [inline tidb-binlog/slave_binlog_proto/proto/binlog.proto]: Align the comment and the lines below?

**Comment by Ryan-Git** [comment]
LGTM

File: tidb-binlog/slave_binlog_proto/proto/binlog.proto (hunk 1 above)
Comment (july2993) [reply tidb-binlog/slave_binlog_proto/proto/binlog.proto]: fixed

**Review by suzaku — APPROVED** [review]
LGTM

**Pull request merged**